create_primers.py

```diff
@@ -0,0 +1,261 @@
+"""Script for creating gene assembly primers.
+
+Primers containing specified mutageneic codons at their center are tiled along a gene.
+
+Edited by Caelan Radford in 2020, from the CodonTilingPrimers code written by Jesse Bloom in 2013 and edited by Adam Dingens Nov 2015 to generate primers of differing lengths to all have a Tm of ~60C.
```

Comment:
Could you add new lines to make this doc string easier to read? In python, lines should technically be no more than 79 characters long.